src/main/java/com/julienviet/pgclient/impl/ArrayTuple.java

```diff
@@ -91,6 +92,13 @@ public Double getDouble(int pos) {
     return null;
   }
 
+  @Override
+  public BigDecimal getBigDecimal(int pos) {
+    if(get(pos) instanceof BigDecimal)
```

Comment:
I think we need also to handle the case where the object is an instanceof Number and convert it